### PR TITLE
feat: add Dark theme

### PR DESCRIPTION
**Pull request by PuneetP16**
Dark Theme: Toggle added

image

### CODE REVIEW

File: src/javascript/darkmodetoggler.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+const disableDarkMode = (e) => {
+	rootEl.style.setProperty("--COLOR-BG", "#e6f0ff");
+	rootEl.style.setProperty("--COLOR-TXT", "#000000");
+	rootEl.style.setProperty("--COLOR-OFFWHITE", "#d3dded");
+	rootEl.style.setProperty("--COLOR-CARD-BG", "white");
+	localStorage.setItem("darkmode", null);
+};
+
+const darkModeClickHandler = (e) => {
```
Comment (muskanbagrecha) [inline src/javascript/darkmodetoggler.js:23]: Neat!

**Comment by muskanbagrecha** [comment]
Loved the UI. Very smooth.

**Review by muskanbagrecha — APPROVED** [review]
Suggestion: On clicking the icons on top, I can see an aqua-like color. I personally didn't like it. You can change it to better suit your theme.

**Comment by sushmita12345** [comment]
Clean code & smooth toggle, looks great.
Looks good to merge

**Review by sushmita12345 — APPROVED** [review]

File: src/javascript/darkmodetoggler.js (hunk 1 above)
Comment (PuneetP16) [reply src/javascript/darkmodetoggler.js:23]: > Suggestion: On clicking the icons on top, I can see an aqua-like color. I personally didn't like it. You can change it to better suit your theme.

Yeah, thanks for mentioning it. I will update it in the next PR!

**Review by iHarryD — APPROVED** [review]
Minimal and smooth UI. Code looks good too. Nice job.

**Comment by yashodaAgarwal** [comment]
Toogle works perfect.

**Pull request merged**